.gitignore

```
@@ -54,3 +54,4 @@ tests/testthat/test\.json
 tests/testthat/batch\.csv/
 tests/testthat/iris-in/
 tests/testthat/iris-out/
+.Rproj.user
```

```suggestion
\.Rproj\.user
```

because the entries in .gitignore are interpreted as regexes